### DISCUSSION TITLE
how to delete an assets folder?

### DISCUSSION THREAD

**Discussion by alisa0920** [requarks/wiki#7254] [Help / Questions]
Hi :
  i want to delete my folder ,but i can not  found any button to delete my folder,like this picture
<img width="697" alt="1" src="https://github.com/user-attachments/assets/6b8644d7-eee7-4e19-90f3-31e2e7b1b37f">

**Comment by fnafox** [comment]
Unfortunately the only way to delete an asset folder is by modifying the database.
```
select * from "assetFolders";
delete from "assetFolders" where id=<id>;
```
